node/silkworm/common/stopwatch.cpp

```diff
@@ -57,7 +57,7 @@ StopWatch::Duration StopWatch::lap_duration() noexcept {
 }
 
 StopWatch::Duration StopWatch::since_start(const TimePoint& origin) noexcept {
-    if (!started_) {
+    if (start_time_ == TimePoint()) {
```

Comment:
Would it make sense to start StopWatch on construction?

Reply:
Yes but with defaulted parameter in constructor. Say you want to create a Stopwatch far from where it is effectively used and pass it around by reference in several functions then the recorded timings wouldn't be accurate.